src/scripts/i18n/inline_and_diff.py

```
@@ -0,0 +1,201 @@
+base_dir = Path(f"src/scripts/i18n/{type_name}")
+base_dir.mkdir(parents=True, exist_ok=True)
+
+# ---------------------- 1. List source files  ---------------------- #
+SUFFIX_MAP = {
+    "components": [".component.ts", ".component.html"],
+    "services": [".service.ts"],
+    "pipes": [".pipe.ts"],
+    "directives": [".directive.ts"],
+}
+
+def find_files(root_dir: str, suffixes: List[str]) -> List[str]:
```

Ruff (0.11.9)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)